components/brave_wayback_machine/pref_names.h

```diff
@@ -6,6 +6,7 @@
 #ifndef BRAVE_COMPONENTS_BRAVE_WAYBACK_MACHINE_PREF_NAMES_H_
 #define BRAVE_COMPONENTS_BRAVE_WAYBACK_MACHINE_PREF_NAMES_H_
 
+// Deprecated. Only used for prefs cleanup.
 inline constexpr char kBraveWaybackMachineEnabled[] =
     "brave.wayback_machine_enabled";
```

Comment:
We could keep this preference if we don't want the logic running at all - like don't show the action button in cases it found a result and page had 404.
cc: @rebron

Reply:
I think we should have a toggle to hide the action button. We've gotten requests to do so for other action buttons like PWA.

Reply:
Added back the settings.